react/components/Conditions/index.js

```diff
@@ -0,0 +1,274 @@
+/* eslint-disable camelcase */
```

Comment:
Is that really necessary?

Reply:
it was just because of the `EXPERIMENTAL_` name. Now it's not necessary anymore. Nice catch.